pkgs/applications/graphics/ImageMagick/7.0.nix

```
@@ -38,6 +38,10 @@ stdenv.mkDerivation {
 
   enableParallelBuilding = true;
 
+  preConfigure = ''
+    sed -i -e 's/\(ac_cv_path_[a-zA-Z]*="\) "\(\$[a-zA-Z]*"\)"/\1\2/' configure # remove breaking quotes
```

```suggestion
    # remove breaking quotes
    sed -i -e 's/\(ac_cv_path_[a-zA-Z]*="\) "\(\$[a-zA-Z]*"\)"/\1\2/' configure
```

Also could this be converted into a patch? This sed is quite complex and we want to notice when the issue is fixed.

---

I have moved this into a much more readable patch and rebased into a single commit.